src/components/deviceMap/regionDetails.js

```diff
@@ -59,7 +59,7 @@ class RegionDetails extends Component {
                 <polygon points="25,2.5 48.5,49.8 2.5,49.8" fill="#fc540a" />
               </svg>
               <div className="alarms">
```

Comment:
nit: className is alarms should that be renamed as well?

Reply:
Okay. I will do right away. Thanks for reviewing.